log/sequencer.go

```diff
@@ -282,8 +280,25 @@ func (s logSequencingTask) update(ctx context.Context, leaves []*trillian.LogLea
 	return nil
 }
 
+type preorderedLogSequencingTask logSequencingTask
+
+func (s *preorderedLogSequencingTask) fetch(ctx context.Context, limit int, cutoff time.Time) ([]*trillian.LogLeaf, error) {
+	// TODO(pavelkalinnikov): Measure latency.
+	leaves, err := s.tx.GetLeavesByRange(ctx, s.treeSize, int64(limit))
```

Comment:
Should probably reword the comment next to `GetLeavesByRange` because it doesn't say returning entries past the tree size is normal for `PREORDERED_LOG`.

Reply:
Done.